src/api/entities/CheckpointSchedule/__tests__/index.ts

```diff
@@ -37,6 +38,7 @@ describe('CheckpointSchedule class', () => {
     remaining = 11;
     nextCheckpointDate = new Date('10/14/2030');
     u64ToBigNumberStub = sinon.stub(utilsConversionModule, 'u64ToBigNumber');
```

Comment:
You shouldn't stub this method, the mock utils are coded in a way that they can be converted normally